docs/language.md

```diff
@@ -287,7 +290,7 @@ Signature | Notes
 `zip(v1: vec[T1], v2: vec[T2], ...): vec[{T1, T2, ...}]` | Only allowed in the `vec` argument of the `for` loop.
 `compare(x: T, y: T)` | Implements a default comparator for `sort`. Expands to `if(x > y, 1, if(x < y, -1, 0))`.
 
-All of these operations can straightforwardly be translated into `for` expressions.
+With the exception of `zip` (which is a special syntactic sugar that isn't expanded from a macro), all of these operations can be translated into `for` expressions.
```

Comment:
it is a little confusing to have `zip` here because the consequences are very different from macros. maybe consider separating it into its own subsection

Reply:
Hmm yeah, they're already discussed in the for loop section, so I just deleted the reference to this here.